Resources/Maps/stationstation.yml

```diff
@@ -2,7 +2,7 @@ meta:
   format: 2
   name: DemoStation
   author: Space-Wizards
-  postmapinit: false
```

Comment:
This is kind of a problem. I didn't make a sane mapper mode yet so you modified and re-saved the map post init. We can't actually use the map file like this.

Reply:
Hmm... How can I fix this? Is there some way I can edit the map pre-init?

Reply:
Yes but IIRC the placement system isn't quite aware of it yet.

You can generate a pre-init map with `addmap <mapID> false` and then load the blueprint into it, and saving wouldn't modify it like this.
But again, IIRC placement isn't aware of it currently so it will still map init things when you place it.